packages/dynamic_layouts/example/test/wrap_example_test.dart

```diff
@@ -23,10 +22,21 @@ void main() {
     expect(find.text('Index 3'), findsOneWidget);
     expect(find.text('Index 4'), findsOneWidget);
 
+    // Material 3 changes the expected layout positioning.
+    final bool usesMaterial3 = (app.theme ?? ThemeData.light()).useMaterial3;
```

Comment:
Yes, I double checked. The slight change in offset is expected.

Reply:
Perfect, thanks for verifying!